db/seeds/activities/stem_learning.rb

```diff
@@ -2602,3 +2602,15 @@
 
   activity.programmes = [secondary_certificate]
 end.save!
+
+Activity.find_by(stem_course_template_no: "690b99ca-1106-ef11-9f8a-002248c635a9")&.destroy
```

Comment:
You should also remove the block that this is deleting as well, as otherwise we are seeding and then immediately destroying every time. We have the & to catch when the find doesn't match in future. 

I would also just add a comment as to why and what we are deleting, just for future reference